packages-content-model/roosterjs-content-model-dom/lib/formatHandlers/segment/fontSizeFormatHandler.ts

```diff
@@ -49,12 +59,18 @@ function normalizeFontSize(fontSize: string, contextFont: string | undefined): s
         fontSize == 'smaller' ||
         fontSize == 'larger' ||
         fontSize.endsWith('em') ||
-        fontSize.endsWith('%')
+        fontSize.endsWith('%') ||
+        fontSize.endsWith('rem')
     ) {
         if (!contextFont) {
             return undefined;
         } else {
-            const existingFontSize = parseValueWithUnit(contextFont, undefined /*element*/, 'px');
+            const existingFontSize = parseValueWithUnit(
```

Comment:
Instead of parsing the value here, for rem, can we just store a pixel value in editorContext? Then here we can directly use it as a number

Reply:
Made the change, let me know if it looks good.